common/djangoapps/adaptive_learning/config/forms.py

```diff
@@ -0,0 +1,40 @@
+"""
+Defines a form for providing validation.
+"""
+import logging
+
+from django import forms
+
+from adaptive_learning.config.models import CourseAdaptiveLearningFlag
+
+from opaque_keys import InvalidKeyError
+from xmodule.modulestore.django import modulestore
+from opaque_keys.edx.locator import CourseLocator
```

Comment:
@tomaszgy Sorry for not catching this earlier, but the import order is off here. It should be [Python Standard Library > Third Party > Current Python Project](https://github.com/timothycrosley/isort#how-does-isort-work), and the imports should be sorted alphabetically within these blocks. You should be able to use `isort` to fix the import order.

Reply:
👍

Reply:
@tomaszgy Did you end up running `isort` on this file? If so, I'm wondering why 

```python
from adaptive_learning.config.models import CourseAdaptiveLearningFlag
```

remained at the top of the list. I would have expected it to be listed last, given that it imports a class from a module that belongs to the "current Python project".